### ISSUE TITLE
ESlint extension stopped working and will not install

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/vscode#26448]
_From @MrSkinny on May 8, 2017 19:48_

VSCode: 1.12.1

I noticed ESLint wasn't running in my project and went to the extensions tab. ESLint was no longer installed.  I searched for it on the marketplace but when I hit "Install", the button changes state for a second and immediately flicks back to "Install".  I am able to run/install other extensions.

Anyone else having this issue?

_Copied from original issue: Microsoft/vscode-eslint#235_

**Comment by dbaeumer** [comment]
_From @MrSkinny on May 8, 2017 20:22_

I've fixed the issue -- despite ESLint not appearing in my extensions list or linting the code in my editor pane, when I went to `~/.vscode/extensions` the folder for the extension still existed.  I deleted it and then tried installing from the marketplace, and this time it worked.  

Other steps taken prior to fixing:-
* Hard reboot
* Updated to Node 6.10.3 
* Reinstalled eslint -g
* Deleted and reinstalled VSC

Throughout, I was able to install, uninstall, and use other existing/new extensions.  

This is on OSX 10.11.6
